.gitignore

```diff
@@ -49,3 +49,6 @@ lib/
 yarn.lock
 package-lock.json
 npm-shrinkwrap.json
+
+# IDE
+.idea
```

Comment:
this can go in your global gitignore rather than in every repo you touch :-)

Reply:
@ljharb True, but on some projects, I like to commit some [IDE configuration](https://intellij-support.jetbrains.com/hc/en-us/community/posts/207656425-Arguments-for-NOT-adding-the-entire-idea-folder-to-gitignore-hgignore). It makes it easier also to share some configuration with other people, and with myself when I switch computers :sweat_smile:.

Reply:
That can be done via dotfiles repos and similar :-)

Reply:
@ljharb I'm not sure of what you mean by "*dotfiles repo*", but if you mean a configuration repository common to all projects (and team), that is a complementary option, but some configurations are still custom to the project and therefore belong in the project itself. I see you even have a file like this here: [`import.sublime-project`](https://github.com/benmosher/eslint-plugin-import/blob/master/import.sublime-project).

But ok, I understand you really don't want to have this added to the `.gitignore`, I don't mind :wink:. I removed it here: https://github.com/adjerbetian/eslint-plugin-import/commit/ba0d05ad1ae5f3e77ed741782a23031713221925